src/Controller/Admin/TalkController.php

```diff
@@ -43,7 +43,10 @@ protected function persistTalkEntity(Talk $talk, Form $newForm): void
                 $submit->setConference($conference);
                 $submit->setTalk($talk);
                 $submit->setStatus(Submit::STATUS_PENDING);
-                $submit->addUser($this->getUser());
```

Comment:
I saw this block of code in another PR an I don't get it. The form component should do it for you. What happens if you delete all the foreach?

Reply:
Talked in slack. It seems legit. I'll look at it later.